uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1002.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1002 {
+
+  /**
+   * Creates the 'other' sickness reason option value.
+   *
+   * @return bool
+   */
+  public function upgrade_1002() {
+    try {
+      $result = civicrm_api3('OptionValue', 'get', [
+        'sequential' => 1,
+        'option_group_id' => 'hrleaveandabsences_sickness_reason',
+        'name' => 'other',
+      ]);
+
+      if ($result['count'] == 0) {
+        civicrm_api3('OptionValue', 'create', [
+          'option_group_id' => 'hrleaveandabsences_sickness_reason',
+          'name' => 'other',
+          'label' => 'Other - Please leave a comment',
+          'value' => 12,
+          'weight' => 11,
+          'is_reserved' => 1,
+          'is_default' => 0,
+          'is_active' => 1
+        ]);
+      }
+    } catch(Exception $e) {
+      // We run all the upgraders during the extension installation, but, during
```

Comment:
Interesting to see about the over-zealous caching. Glad you found the source of it :slightly_smiling_face: